tests/loggers/test_cometml_logger.py

```diff
@@ -76,16 +77,42 @@ def test_comet_ml_log_image_saves_images(images: torch.Tensor, channels_last: bo
     assert actual_num_images == expected_num_images
 
 
-@pytest.mark.parametrize(
-    'images,channels_last',
-    [
-        (torch.rand(32, 0), False),  # Has zero in dimension.
-        (torch.rand(4, 4, 8, 32, 32), False),  # > 4 dim.
-        ([torch.rand(4, 32, 32, 3)], True),
-    ])  # sequence > 3 dim.
-def test_comet_ml_log_image_errors_out(comet_logger, images, channels_last):
-    with pytest.raises(ValueError):
-        comet_logger.log_images(images, channels_last=channels_last)
+@pytest.mark.parametrize('images,masks', [(torch.randint(0, 256, (32, 32, 3)), {
```

Comment:
Can you add a test with not channels last as well?

Reply:
done